src/migrations/00037-add-contract-release.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "release" 
+ADD COLUMN IF NOT EXISTS "contract" TEXT NULL;
```

Comment:
I saw we are using `TEXT` in the migrations instead of JSON, despite the sbvr type being JSON. Any reason for that?

Reply:
The SBVR JSON type predates the binary JSON types (and possibly the JSON type altogether?) so it uses TEXT - it'd be nice to move over to more closely match what would be expected but in practical terms we don't support any of the special things the JSON/JSONB types enable so it doesn't really affect anything